### PR TITLE
add clarification to installers note

### LINKED ISSUE #3956: Feedback: 'Installers'
Feedback for 'Installers' https://docs.particular.net/nservicebus/operations/installers

Location in GitHub: https://github.com/Particular/docs.particular.net/blob/master/nservicebus/operations/installers.md

ping @Particular/nservicebus-maintainers

The sentence 

> Consider running the installers only during the initial setup using elevated permissions

surprised me as it is seldom the case that the application can change permission levels. Is the user advised to make platform dependent system service calls, or does NServiceBus offer a platform independent abstraction for that? Consequently, a reference to some text explaining how this is being done would be helpful.

### PR DESCRIPTION
**Pull request by adamralph**
Closes #3956

### CODE REVIEW

File: nservicebus/operations/installers.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Installers ensure that endpoint-specific artifacts (e.g. queues, directories, da
 
 ## Running installers
 
-NOTE: Running installers requires the process to run with permissions to create/delete tables, queues, folders and other resources. It is recommended to apply the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege) and prevent running an endpoint process with too many permissions. Consider running the installers only during the initial setup, using elevated permissions.
+NOTE: Running installers requires the endpoint process to run with permissions to create/delete tables, queues, folders and other resources. It is recommended to apply the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege) and avoid running a process with redundant permissions. Consider running installers only during initial deployment, using elevated permissions. For example: start an endpoint process manually, as an administrator or elevated user, before installing it as a service. The alternative to using installers is to create the required resources before the endpoint is run. The method varies for each transport or persistence package. For more information, see [operations](/nservicebus/operations).
```
Comment (bording) [inline nservicebus/operations/installers.md]: Not sure "redundant" is correct here.

Comment (bording) [reply nservicebus/operations/installers.md]: "Consider running installers only during initial deployment, using elevated permissions"

Does this need a comma?

Comment (bording) [reply nservicebus/operations/installers.md]: "start an endpoint process manually, as an administrator or elevated user, before installing it as a service."

Don't think any of those commas are needed.

Comment (bording) [reply nservicebus/operations/installers.md]: "The method varies for each transport or persistence package."

Feels like it needs something like "The method to do this varies"

Comment (adamralph) [reply nservicebus/operations/installers.md]: The more I read it, the more I realise that I'm battling against the current structure. I'll attempt a re-write, to see if it comes out better.

File: nservicebus/operations/installers.md [hunk 2]
```diff
@@ -11,11 +11,11 @@ redirects:
...
+NOTE: Installers require permission to administer resources such as database tables, queues, or directories. Following the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege) it is recommended to run an endpoint with these elevated permissions only during initial deployment. For example: an endpoint can be started manually as an administrator or elevated user to allow installers to run. The required resources will be ready as soon as the endpoint has successfully started. The endpoint can then be shut down and installed as a service. The service can be configured to run as a regular user which does not have permission to modify resources. The alternative to using installers is to create the required resources before the endpoint is run. The method of doing this varies for each transport or persistence package. For more information, see [operations](/nservicebus/operations).
```
Comment (bording) [inline nservicebus/operations/installers.md]: " Installers require permission"

Should "permission" be "permissions"?

Comment (bording) [reply nservicebus/operations/installers.md]: Needs a comma after "(https://en.wikipedia.org/wiki/Principle_of_least_privilege)"

Comment (bording) [reply nservicebus/operations/installers.md]: "For example:"

Should that just be a comma instead of a colon?

**Comment by bording** [comment]
>I'm also wondering whether this is getting too big for a note, and needs to be a section of it's own. It could still have a shorter note, which alerts the reader, and links to the section.

I think moving this to it's own section makes sense.

File: nservicebus/operations/installers.md (hunk 2 above)
Comment (bording) [reply nservicebus/operations/installers.md]: "The endpoint can then be shut down and installed as a service. The service can be configured to run as a regular user which does not have permission to modify resources."

Better as 

The endpoint can then be shut down and installed as a service, which can run as a regular user that does not have permission to modify resources."  ?

**Comment by adamralph** [comment]
I've moved the installers to a new section titled "When to run installers".

**Pull request merged**